tests/test_basic.py

```diff
@@ -22,6 +25,19 @@ def test_dict__plain_match():
     assert {"a": 1, "b": 2} == DictContains({})
 
 
+def test_anyof():
```

Comment:
Well, it should be vice versa =)

Actual value should be equal to one of "AnyOf". Not "AnyOf" is equal to one of the actual.

I.e. it should do:
```(python)
assert 1 == AnyOf(1, 2, 3)
```